test/functional/apps/dashboard_elements/controls/controls_callout.ts

```diff
@@ -25,12 +29,24 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     describe('callout visibility', async () => {
       before(async () => {
         await dashboard.gotoDashboardLandingPage();
+
         await dashboard.clickNewDashboard();
         await timePicker.setDefaultDataRange();
         await dashboard.saveDashboard('Test Controls Callout');
       });
 
       describe('does not show the empty control callout on an empty dashboard', async () => {
+        before(async () => {
+          const panelCount = await dashboard.getPanelCount();
+          if (panelCount > 0) {
+            const panels = await dashboard.getAllPanels();
+            for (const panel of panels) {
+              await dashboardPanelActions.removePanel(panel);
+            }
+            await dashboard.clickQuickSave();
+          }
+        });
```

Comment:
Because you can't guarantee the order that inner test suites run in, I was running in to an issue when the tests below (which add visualizations to the dashboard) were running **before** the empty dashboard tests, so I needed to ensure that the dashboard was empty here to get things to reliably pass

Reply:
Makes sense!